### ISSUE TITLE
pdffiller-stream doesn't seem to work with accents. 

### ISSUE THREAD

**Issue by kiskalaci** [Sparticuz/pdffiller-stream#11]
My issue is that, when I use accents within the map such as {Név: "Patient2"}, the form fields will not get filled out. Whereas when I don't use any accents, then it works as intended. I'm I missing something or is this a bug . 

Initially I was using the original pdffiller package and this issue was not present.

**Comment by Sparticuz** [comment]
@klaczi Is the issue present when using jasonphillips/pdffiller-stream? Are you saying the field doesn't get filled out with the accent? Or you are using a regex to map the Field names to your json object?

I haven't quite gotten my fork up to standards for public release yet. That's why the npm package is under my namespace, still testing and moving a lot of code around.

**Comment by kiskalaci** [comment]
I've only tried your fork atm. It does get filled out, but with random text and characters which I could no comprehend. Currently I'm trying out hummus js but pdffiller - stream seemed interesting. Thanks for the fast reply :) !

**Comment by Sparticuz** [comment]
I see the same issue, but I thought it might be from the embedded font in
the PDF not having those characters. Haven't had a chance to embed a font
in a pdf to test it yet.

**Comment by gitowiec** [comment]
Here is my way to deal with accents
https://stackoverflow.com/questions/65037718/what-is-the-perfect-way-to-fill-acroforms-using-cli-linux-tool-or-nodejs-module
TL;DR;
You have to pass `need_appearances` to `psdtk` which excludes the use of `flatten` - sink or swim :(

**Comment by Sparticuz** [comment]
Ok, I'll look into adding that as an option soon.
